src/app/myworkflows/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -75,7 +91,23 @@ export class SidebarAccordionComponent implements OnInit, OnChanges {
   trackByOrgWorkflowObject(index: number, orgWorkflowObject: OrgWorkflowObject<Workflow>) {
     return orgWorkflowObject.sourceControl + '/' + orgWorkflowObject.organization;
   }
+
   openGitHubAppsLogs(organization: string) {
     this.dialog.open(GithubAppsLogsComponent, { width: bootstrap4largeModalSize, data: organization });
   }
+
+  /**
+   * If GitHub app tool, apply tool style (blue theme)
+   */
+  isGitHubAppTool(): boolean {
+    let isAppTool: boolean = false;
+    this.entryType$
+      .subscribe((value) => {
+        if (value === this.EntryType.Tool) {
```

Comment:
Does this work with `this.EntryType.AppTool`?

Reply:
mm no it doesn't, and when I print "value" it says it's a tool